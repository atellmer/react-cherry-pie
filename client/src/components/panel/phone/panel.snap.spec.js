import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';

import TmPanelPhone from './index';


const props = {
  dialogs: []
};

test('Jest: TmPanelPhone (Snapshot)', () => {
  const shallowRenderer = ReactTestUtils.createRenderer();

  const result = shallowRenderer.render(<TmPanelPhone {...props}/>);
  expect(result).toMatchSnapshot();
});
