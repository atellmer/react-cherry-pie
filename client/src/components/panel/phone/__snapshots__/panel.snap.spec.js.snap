exports[`test Jest: TmPanelPhone (Snapshot) 1`] = `
<div
  className={undefined}>
  <Scrollbars
    autoHeight={false}
    autoHeightMax={200}
    autoHeightMin={0}
    autoHide={true}
    autoHideDuration={200}
    autoHideTimeout={1000}
    className={undefined}
    hideTracksWhenNotNeeded={false}
    renderThumbHorizontal={[Function]}
    renderThumbVertical={[Function]}
    renderTrackHorizontal={[Function]}
    renderTrackVertical={[Function]}
    renderView={[Function]}
    tagName="div"
    thumbMinSize={30}
    universal={false}>
    
  </Scrollbars>
</div>
`;
